libcloud/test/storage/test_scaleway.py

```diff
@@ -0,0 +1,56 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import sys
+import unittest
+
+from libcloud.common.types import LibcloudError
+from libcloud.storage.drivers.scaleway import ScalewayStorageDriver
+from libcloud.storage.drivers.scaleway import ScalewayConnectionAWS4
+
+from libcloud.test.secrets import STORAGE_S3_PARAMS
+
+
+class ScalewayStorageDriverTestCase(unittest.TestCase):
```

Comment:
Should inherit base s3 tests class here so it verifies that all the base S3 driver related functionality works as expected. IIRC, some existing drivers such as one of the Aurora objects already do that (libcloud/test/storage/test_aurora.py).

Reply:
Wilco